development/src/components/userLogin_comp.js

```diff
@@ -65,102 +74,126 @@ function UserLoginComp() {
-            </Link>
-          </p>
-        </div>
+              <div className="flex flex-row justify-between gap-3">
+                <button
+                  onClick={(e) => {
+                    e.preventDefault();
+                    signInWithGoogle();
+                  }}
+                  className="flex flex-row flex-nowrap justify-center gap-2 bg-[#DCDCE5] dark:bg-[#363647] items-center p-3 rounded-md w-full shadow-md"
+                >
+                  <Image
+                    src={googleIcon}
+                    alt="google_icon"
+                    className="w-6 h-auto"
+                  />
+                  <p className="text-[14px]">Create Account with Google</p>
+                </button>
+                <button
+                  onClick={(e) => {
+                    e.preventDefault();
+                    signInWithGithub();
+                  }}
+                  className="flex flex-row flex-nowrap justify-center gap-2 bg-[#DCDCE5] dark:bg-[#363647] items-center px-3 rounded-md w-full shadow-md"
+                >
+                  <Image
+                    src={theme === "dark" ? github_darkMode : github_lightMode}
+                    alt="google_icon"
+                    className="w-6 h-auto"
+                  />
+                  <p className="text-[14px]">Create Account with Github</p>
+                </button>
+              </div>
+            </div>
+
+            <div className="flex items-center text-center">
+              <div className="border-b-2 border-gray-200 w-full relative flex justify-center"></div>
+              <p className="flex justify-center w-1/6">OR</p>
+              <div className="border-b-2 border-gray-200 w-full relative flex justify-center"></div>
+            </div>
+
+            <div className="space-y-8">
+              <div>
+                <label htmlFor="email">Email Address</label>
+                <input
+                  type="email"
+                  name="email"
+                  id="email"
+                  className="dark:bg-[#363647] border-[1px] p-3 rounded-lg bg-gray-100 w-full"
+                  placeholder="Enter Email Address"
+                  onChange={emailChange}
+                  value={emailAddress}
+                />
+                {emailError && (
+                  <p className="text-sm text-red-500">{emailError}</p>
+                )}
+              </div>
+
+              <div>
+                <label htmlFor="password">Password</label>
+                <PasswordToggle
+                  inputId="password"
+                  placeholder="Enter password"
+                  handleInputChange={passwordChange}
+                  inputValue={password}
+                />
+                {passwordError && (
+                  <p className="text-sm text-red-500">{passwordError}</p>
+                )}
+                <button
+                  className="float-right font-bold text-[#5F5BD7]"

```

Comment:
remove the `font-bold` from the forgot password

Reply:
Done